core-web/apps/dotcms-ui/src/app/view/components/dot-api-link/dot-api-link.component.html

```diff
@@ -1,6 +1,6 @@
 <ng-container *ngIf="link">
-    <a class="p-button-sm p-button-outlined" [href]="link" [title]="link" pButton target="_blank">
-        <i class="material-icons">link</i>
-        API
+    <a class="p-button-sm p-button-text" [href]="link" [title]="link" pButton target="_blank">
```

Comment:
What the `pButton` directive do to this? doesn't work?

Reply:
it just add the primeNG classes